Assets/Scripts/InteractWithLookedAt.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+/// <summary>
+/// Detects when the player presses the Interact button while looking at an IInteractive 
+/// and then calls that IInteractive's InteractWith method.
+/// </summary>
```

Comment:
need line of white space

Reply:
We talked about it and decided since it was a consistent style, it was fine.